packages/orchestration/src/exos/remote-chain-facade.js

```diff
@@ -125,8 +125,6 @@ const prepareRemoteChainFacadeKit = (
         makeAccount() {
           return asVow(() => {
             const { remoteChainInfo, connectionInfo } = this.state;
-            const stakingDenom = remoteChainInfo.stakingTokens?.[0]?.denom;
-            if (!stakingDenom) throw Fail`chain info lacks staking denom`;
```

Comment:
I can't recall why this was a check. It doesn't seem we rely on `stakingDenom` (anymore?)

Reply:
I'm struggling to understand what this test is trying to achieve / how we can replace it: 
https://github.com/Agoric/agoric-sdk/blob/fc802adc06082eb0618f4a2d58d91ac380512352/packages/orchestration/test/facade-durability.test.ts#L83_L122

Reply:
IIRC having a stakingDenom was a proxy for some other guarantee about the chain, but I don't remember what that is and it seems like it's not necessary (if it ever was) because the other tests are passing.

Reply:
I think I'm now remembering it was a proxy for which chains can make an ica / have staking actions.

It reminds me to create a ticket for `CosmosChainInfo['icaEnabled']` (this is true for all current chains in `KnownChains`). This is where we'd perform the check.

For something like `delegate: never`, we can continue rely on the `CosmosChainAccountMethods` type that looks for stakingTokens.

Reply:
It seems to date back to...
 - #9532

I used `git log --reverse -S'chain info lacks staking denom' -- src/exos/remote-chain-facade.js` to find the 1st version it appeared in.